Document/0x05b-Basic-Security_Testing.md

```diff
@@ -65,6 +65,7 @@ Once you've configured the network and established a connection between the test
 
 - The proxy must be [configured to point to the interception proxy](https://support.portswigger.net/customer/portal/articles/1841101-Mobile%20Set-up_Android%20Device.html "Configuring an Android Device to Work With Burp").
 - The [interception proxy's CA certificate must be added to the trusted certificates in the Android device's certificate storage](https://support.portswigger.net/customer/portal/articles/1841102-installing-burp-s-ca-certificate-in-an-android-device "Installing Burp's CA Certificate in an Android Device"). The location of the menu used to store CA certificates may depend on the Android version and Android OEM modifications of the settings menu.
+- Some application (e.g. the [Chrome browser](https://bugs.chromium.org/p/chromium/issues/detail?id=475745)) may show `NET::ERR_CERT_VALIDITY_TOO_LONG` errors, if the leaf certificate happens to have a validity extending a certain time (39 months in case of Chrome). This happens if the default Burp CA certificate is used, since the Burp Suite issues leaf certificates with the same validity as its CA certificate. You can circumvent this by creating your own CA certificate and import it to the Burp Suite, as explained in [this](https://blog.nviso.be/2018/01/31/using-a-custom-root-ca-with-burp-for-inspecting-android-n-traffic/) blog post.
```

Comment:
I have not yet encountered this bug on mobile apps. do you have a sample of it (e.g. is there a specific version of android/chrome where this happens?)?

Reply:
> Can you please add hover-titles to links? `[title](link "hover-title")`

Added in [Commit ba75a71](https://github.com/OWASP/owasp-mstg/commit/ba75a7154ba3b3e31c2ed1d601ba854260255c41)


> I have not yet encountered this bug on mobile apps. do you have a sample of it (e.g. is there a specific version of android/chrome where this happens?)?

I just reproduced the error yesterday with my Google Pixel 2XL running Android version `8.1.0` and Chrome version `72.0.3626.105`. But I guess it would help if you try it with your device as well?!

Follow these steps (see also [the blog post I mentioned](https://blog.nviso.be/2018/01/31/using-a-custom-root-ca-with-burp-for-inspecting-android-n-traffic/)):
- Open Burp and use `Regenerate CA certificate` (make sure to make a backup copy of your CA certificate before)
- Export the Burp CA certificate in DER format and push it to your device
- Use a way to add the CA certificate to the system's CA trust store
- Setup a way to redirect the traffic
- Open Chrome browser and browse to any site will show this error